octoprint_octobullet/__init__.py

```diff
@@ -56,24 +156,30 @@ def on_settings_save(self, data):
 		octoprint.plugin.SettingsPlugin.on_settings_save(self, data)
 
 		import threading
-		thread = threading.Thread(target=self._connect_bullet, args=(self._settings.get(["access_token"]),
+		thread = threading.Thread(target=self._connect_bullet, args=(self._settings.get(["quiet_minutes"]),
```

Comment:
This will produce errors. You have added a parameter to the call to `self._connect_bullet` that that method will neither accept nor need.